internal/utils.go

```diff
@@ -51,17 +44,19 @@ func FormatCacheSize(size_in_bytes float64) string {
 }
 
 func PrettyPrintCacheList(caches []types.ActionsCache) {
-	width, _, _ := getTerminalWidth(os.Stdout)
-	width = int(math.Max(float64(width), 100))
-	sizeWidth := SIZE_COLUMN_WIDTH             // hard-coded size as the content is scoped
-	timeWidth := LAST_ACCESSED_AT_COLUMN_WIDTH // hard-coded size as the content is scoped
-	keyWidth := int(math.Floor(0.65 * float64(width-SIZE_COLUMN_WIDTH-LAST_ACCESSED_AT_COLUMN_WIDTH)))
-	refWidth := int(math.Floor(0.20 * float64(width-SIZE_COLUMN_WIDTH-LAST_ACCESSED_AT_COLUMN_WIDTH)))
+	terminal := ghTerm.FromEnv()
+	w, _, _ := terminal.Size()
+	tp := ghTableprinter.New(os.Stdout, true, w)
 
 	for _, cache := range caches {
-		var formattedRow string = getFormattedCacheInfo(cache, keyWidth, sizeWidth, refWidth, timeWidth)
-		fmt.Println(formattedRow)
+		tp.AddField(cache.Key)
+		tp.AddField(FormatCacheSize(cache.SizeInBytes))
+		tp.AddField(cache.Ref)
+		tp.AddField(lastAccessedTime(cache.LastAccessedAt))
+		tp.EndRow()
 	}
+
+	_ = tp.Render()
 }
 
 func PrettyPrintTrimmedCacheList(caches []types.ActionsCache) {
```

Comment:
Is this method still used?

Reply:
If not then probably CI should have thrown an error about unused code. Can we enable that if go build allows this feature?

Reply:
Ah yes